src/ui/components/toolbar/components/toolbar-color-picker.vue

```diff
@@ -14,7 +14,7 @@
                 v-if="!this.options.preventNone"
                 @click="select({value: 'transparent'})">
                 <div class="c-palette__item"></div>
-                No fill
```

Comment:
IMO, The `options.type` property is exposing customization that is out of scope for our toolbar components.  This should use a generic label (i.e. `transparent` or `none`) and the type property should be removed from the color picker options.

Reply:
If we don't want to expose that customization at that level, I will remove the type property and use "None" as a generic label.

Reply:
Yes, please.